packages/app/lib/modular/index.js

```diff
@@ -28,7 +28,7 @@ export function deleteApp(app) {
  * Registers a library's name and version for platform logging purposes.
   @param {string} libraryKeyOrName - library name or key.
   @param {string} version - library version.
-  @param {string | null} variant - library variant.
+  @param {string | null} variant - library variant. Optional.
```

Comment:
the question mark indicates it is maybe missing so is maybe undefined, which is distinct from null (null is technically an actual primitive value, undefined is the absence of a value) - so I think this is important also, if this change goes in:

```suggestion
  @param {string | undefined} variant - library variant. Optional.
```